src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -762,7 +762,7 @@ private async Task<(bool Success, string ErrorMessage)> SetTriggersAsync(string
                 string jwtToken = JwtTokenHelper.CreateToken(DateTime.UtcNow.AddMinutes(5));
                 request.Headers.Add(ScriptConstants.SiteTokenHeaderName, jwtToken);
 
-                if (_environment.IsManagedAppEnvironment())
+                if (_environment.IsManagedAppEnvironment() || _environment.IsConnectedAppEnvironment())
```

Comment:
So it looks like your approach was to add this additional check 1:1 in every place where we were already checking for managed environment. Can we add an IsAnyContainerAppsEnvironment() check that encapsulates these and use that everywhere. This'll be less error prone and easier to manage.

Reply:
fixed your suggestion. Thanks